src/test/runTest.ts

```diff
@@ -35,7 +35,8 @@ async function main() {
 		// in the home dir.
 		const extensionTestsEnv = {
 			HOME: tmpHomeDir,  // eslint-disable-line @typescript-eslint/naming-convention
-			INMANTA_LANGUAGE_SERVER_PATH: process.env.INMANTA_LANGUAGE_SERVER_PATH  // eslint-disable-line @typescript-eslint/naming-convention
+			INMANTA_LANGUAGE_SERVER_PATH: process.env.INMANTA_LANGUAGE_SERVER_PATH,  // eslint-disable-line @typescript-eslint/naming-convention,
+			INMANTA_COMPILER_CACHE: "False" // eslint-disable-line @typescript-eslint/naming-convention,
```

Comment:
Even better than the Jenkinsfile.

Reply:
But it seems it doesn't work :(

Reply:
The LS tests run against multiple versions of the product and should be compatible with all of them, so it shouldn't be required there.

Reply:
(I added a comment that didn't make sense and deleted it right after. You probably got a notification, perhaps I should have kept it)

Reply:
No issue, I've done the same multiple times already :p